pages/icons.js

```diff
@@ -12,6 +12,7 @@ export default function Icons() {
   const [notFound, setNotFound] = useState();
   const [threeOrMore, setThreeOrMore] = useState();
 
+  const popularIcons = ["FaGithub", "FaTwitter", "FaLinkedin", "FaGit"];
```

Comment:
Wouldn't it make sense to fetch this value instead of hardcoding it 🤔

Reply:
We can fetch the value as well but by what parameter do we choose which icons are popular.
So hardcoding the popular icons seems more clean

Reply:
the number of people who use that icon?

```js
const profiles = await (fetch('https://linkfree.eddiehub.io/api/users').then(res => res.json()))

let links = []

profiles.forEach(profile => links = [...links, ...profile.links)

const icons = links.map(link => link.icon)

const sortedIcons = icons.sort((a,b) => icons.filter(i === a).length - icons.filter(i === b).length)

// now just pick the top n number of icons and label them as popular
// P.S. This code might need some debugging I just wrote it here in github
```

Reply:
Ohh I did not knew about  the API....It looks good Krish

Reply:
But don't you think this might be a overkill as the popular icons won't change frequently and we would be getting the same icons as popular .
It would be more efficient to have hard coded popular icons than having API calls rendering the same result

Reply:
It is a great idea Krish, but I am keen to keep things simple and fast for now, let's keep it hard coded for now, and we can expand on this in the future if needed